rpm/README.md

```diff
@@ -21,23 +21,23 @@ and enhancements that were not available until after the software release.
    This RPM will only be installed if the SAT CFS configuration layer has been
    applied.
 
-   ```
+   ```bash
    rpm -q docs-sat
    ```
 
 2. Download and upgrade the latest documentation RPM.
 
    For example, to download and upgrade to the latest SAT 2.4 documentation RPM:
 
-   ```
+   ```bash
```

Comment:
These look fine with `bash` highlighting though I'm thinking for consistency they should be `screen`.